tgui/packages/tgui/interfaces/CrewMonitor.js

```diff
@@ -145,7 +145,9 @@ const CrewMonitorDataView = (_properties, context) => {
                   cm.area + ' (' + cm.x + ', ' + cm.y + ')'
                 )
               ) : (
-                'Not Available'
+                <Box inline color='grey'>
+                  {'Not Available'}
```

Comment:
```suggestion
                  Not Available
```
this should compile, if it does can you please use this format instead?